src/main/resources/liquibase/crop_changelog/16_3_0.xml

```diff
@@ -0,0 +1,133 @@
+			<sqlCheck expectedResult="0">
+				SELECT COUNT(*)
+					FROM information_schema.routines
+						WHERE routine_schema = DATABASE() AND specific_name = 'deleteTermIdFromProjectPropTable';
+			</sqlCheck>
+		</preConditions>
+		<comment>Create procedure deleteTermIdFromProjectPropTable</comment>
+		<sql dbms="mysql" splitStatements="false">
+			CREATE PROCEDURE deleteTermIdFromProjectPropTable(termId int)
+			BEGIN
+				DECLARE projectId INT;
+				DECLARE done INT DEFAULT FALSE;
+				DECLARE cur1 CURSOR FOR SELECT project_Id FROM projectprop WHERE variable_id = 8269 order by rank;
+				DECLARE CONTINUE HANDLER FOR NOT FOUND SET done = TRUE;
+
+				OPEN cur1;
+					read_loop: LOOP FETCH cur1 INTO projectId;
+						set @newRank = 0;
+						IF done THEN
+							LEAVE read_loop;
+						END IF;
+
+						DELETE FROM projectprop WHERE project_id = projectId AND variable_id = termId;
+						UPDATE projectprop set rank= @newRank:=@newRank+1 WHERE project_id = projectId order by rank;
+
+					END LOOP read_loop;
+				CLOSE cur1;
+			END
+		</sql>
+	</changeSet>
+
+	<changeSet author="diego" id="v16.3.0-3">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT COUNT(*)
+					FROM information_schema.routines
+						WHERE routine_schema = DATABASE() AND specific_name = 'deleteTermIdFromProjectPropTable';
+			</sqlCheck>
+		</preConditions>
+		<comment> Delete TermId 8269 (StockId) from projectprop table</comment>
+		<sql dbms="mysql" splitStatements="false">
+			call deleteTermIdFromProjectPropTable(8269);
+		</sql>
+	</changeSet>
+
+	<changeSet author="diego" id="v16.3.0-4">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT COUNT(*)
+					FROM information_schema.routines
+						WHERE routine_schema = DATABASE() AND specific_name = 'deleteTermIdFromProjectPropTable';
+			</sqlCheck>
+		</preConditions>
+		<comment>Drop procedure deleteTermIdFromProjectPropTable</comment>
+		<sql dbms="mysql" splitStatements="false">
+			DROP PROCEDURE deleteTermIdFromProjectPropTable;
+		</sql>
+	</changeSet>
+
+	<changeSet author="diego" id="v16.3.0-5">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT COUNT(*) > 0
+					FROM Stockprop WHERE type_id = 8269;
+			</sqlCheck>
+		</preConditions>
+		<comment>Delete TermId 8269 (StockId) from stockprop table</comment>
+		<sql dbms="mysql" splitStatements="true">
+			DELETE FROM Stockprop WHERE type_id = 8269;
+		</sql>
+	</changeSet>
+
+	<changeSet author="diego" id="v16.3.0-6">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT COUNT(*) > 0 FROM cvterm_relationship WHERE subject_id = 8269;
+			</sqlCheck>
+		</preConditions>
+		<comment>Delete TermId 8269 (StockId) from cvterm_relationship and cvterm tables</comment>
+		<sql dbms="mysql" splitStatements="true">
+			DELETE FROM cvterm_relationship WHERE subject_id = 8269;
+			DELETE FROM cvterm WHERE cvterm_id = 8269;
```

Comment:
Should this be done at the end? I think in this case all this can be done in only one changeset asking if the cvterm_id = 8269 and name = 'StockId' exists